testing/src/cypress/e2e/02-institution/05-manage-designations.cy.ts

```diff
@@ -1,30 +1,161 @@
+    manageDesignationObject
+      .designationDetailsText()
+      .scrollIntoView()
+      .should("be.visible");
+    manageDesignationObject
+      .designatedLocationsText()
+      .scrollIntoView()
+      .should("be.visible");
+    manageDesignationObject
+      .signingOfficersText()
+      .scrollIntoView()
+      .should("be.visible");
+    manageDesignationObject
+      .agreementDocumentationText()
+      .scrollIntoView()
+      .should("be.visible");
+    manageDesignationObject
+      .bcPolicyManualHyperlinkText()
+      .scrollIntoView()
+      .should("be.visible")
+      .should(
+        "have.attr",
+        "href",
+        "https://studentaidbc.ca/sites/all/files/school-officials/policy_manual.pdf"
+      );
+    manageDesignationObject
+      .administrationManualHyperlinkText()
+      .scrollIntoView()
+      .should("be.visible")
+      .should(
+        "have.attr",
+        "href",
+        "https://studentaidbc.ca/sites/all/files/school-officials/admin_manual.pdf"
+      );
+    manageDesignationObject
+      .informationSharingAgreementHyperlinkText()
+      .scrollIntoView()
+      .should("be.visible")
+      .should(
+        "have.attr",
+        "href",
+        "https://studentaidbc.ca/sites/all/files/school-officials/Schedule_D_Designation_Agreement.pdf"
+      );
+    manageDesignationObject
+      .designationAcknowledgementsText()
+      .scrollIntoView()
+      .should("be.visible");
+  });
+
+  it('Verify that user redirect manage locations when clicked on "manage locations" hyperlink', () => {
+    manageDesignationObject.manageLocationsHyperlink().click({ force: true });
+    manageInstitutionObject.manageLocations().should("be.visible");
+    cy.url().should("contain", "/manage-locations");
+  });
+
+  it('Verify that user redirect manage users when clicked on "manage users" hyperlink', () => {
+    manageDesignationObject.manageUsersHyperlink().click({ force: true });
+    manageInstitutionObject.manageUsers().should("be.visible");
+    cy.url().should("contain", "/manage-users");
+  });
+});
+
+describe("Request Designation", () => {
+  before(() => {
+    institutionHelperActions.loginIntoInstitutionSingleLocation();
+  });
+
+  beforeEach(() => {
+    dashboardInstitutionObject.manageInstitutionButton().click();
+    manageDesignationObject.manageDesignationButton().click();
+    manageDesignationObject.requestDesignationButton().click();
+  });
+
+  it("Clicking on manage locations should navigate to locations page", () => {
+    manageDesignationObject.manageLocationsHyperlink().click();
+    dashboardInstitutionObject.allLocationsText().should("be.visible");
+  });
+
+  it("Clicking on cancel should navigate back to manage designations page", () => {
+    manageDesignationObject.cancelAgreementButton().scrollIntoView().click();
+    manageDesignationObject.designationAgreementsText().should("be.visible");
+  });
+
+  it.skip("Request a new designation", () => {
```

Comment:
Why is this skipped, im sure there must be a reason. 

Adding a todo comment will help.

Reply:
Intentionally skipped until we have control over the data that is created. Since we can have one `request for designation` in progress. Will add a TODO comment